File: translation_output/deepseekcoder/codenet/python/java/p02719.java
<｜begin▁of▁sentence｜>You are an expert Python programmer.Your task is to write a Python function to solve a programming problem.

### Instruction:
Translate the following python code to java and enclose your solution inside ```java```:
```
n, k = map(int, input().split())

rot = n//k
x = n

if k == 1 or n == k or n%k == 0:
  x = 0
elif rot >= 1:
  x = n - k*rot
  
t = x%k

print(min(t, abs(t-k)))
```

### Response:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int k = scanner.nextInt();

        int rot = n/k;

        int x = n;

        if (k == 1 || n == k || n%k == 0) {
            x = 0;
        } else if (rot >= 1) {
            x = n - k*rot;
        }

        int t = x%k;

        System.out.println(Math.min(t, Math.abs(t-k)));
    }
}
```
<|EOT|>

File: translation_output/deepseekcoder/codenet/java/python/p02719.py
<｜begin▁of▁sentence｜>You are an expert Python programmer.Your task is to write a Python function to solve a programming problem.

### Instruction:
Translate the following java code to python and enclose your solution inside ```python```:
```
import java.util.*;
import java.io.*;
import java.math.*;
 
public class p02719 {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        long n = sc.nextLong();
        long k = sc.nextLong();
        long ans = Math.min(n%k, k-(n%k));
        System.out.println(ans);
    }

}
```

### Response:
```python
import sys

def main():
    n, k = map(int, sys.stdin.readline().split())
    ans = min(n % k, k - (n % k))
    print(ans)

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads two long integers from the standard input, calculates the minimum of the remainder of `n` divided by `k` and `k` minus the remainder of `n` divided by `k`, and prints the result.
<|EOT|>
